tests/browser/pe/program_enrollments.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+# Create A New Course
+
+#     Click Page Button       Next
+#     ${account_id} =  API Create Specific Organization Account      Test
+
+#      &{account} =  API Create Organization Account   
+#      ${course} =  API Create And Return Specific Course     ${account_id}   History
+
+
+
+Verify HEDA Settings
+    Go To Heda Settings
+    Wait Until Element Is visible	        //a[contains(text(),'Affiliations')]
+    Click Link				                //a[contains(text(),'Affiliations')]
+
+    # Default Account Model
+    #Element Should be Visible               //span[contains(@class,'default-account-model-record-type-output-text')]
+
+    #Go into Edit Mode - Note:  we are in Affiliations/Settings
+    Click Button                            Edit
+
+    #Wait Until Element Is visible           //label[@class='slds-checkbox']//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']//following-sibling::span[@class='slds-checkbox--faux']
+    Wait Until Element Is visible           (//label[@class='slds-checkbox'])//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']
```

Comment:
We need to move all the hardcoded locators to locators.py file and call it from there.

Reply:
that's for phase 2

Reply:
I created the functions needed to pull in locators from locators file, and moved hardcoded out